content/blog/a-summary-of-the-july-retrospectives/index.md

```diff
@@ -0,0 +1,78 @@
+
+Several changes went in ahead of the July CPU that introduced delays during the release.  2 of the changes are enhancements as part of our Secure Software Development Framework [(SSDF) activities](https://github.com/adoptium/adoptium/issues/120), [GPG signing](https://blog.adoptium.net/2022/07/gpg-signed-releases/) and the [addition of SBOM artifacts](https://github.com/adoptium/temurin-build/issues/2900). Both of these features introduce new artifacts that can be downloaded alongside of the JDK binaries.  
+
+During the release, there were some initial missteps when not all of the new artifacts were published.  While this was easily rectified, it prompted the team to add checks for ensuring we are publishing the expected set of artifacts.   
+
+A [Windows compiler upgrade](https://github.com/adoptium/temurin-build/pull/2992) was also attempted prior to the release, there were unwanted side-effects, like missing libraries.  These went unnoticed because we had no tests for verifying the full set to be produced, and other testing did not spot the missing libraries either, because our test machines have the compiler installed in order to compile native test material.  The immediate action was to [revert the change](https://github.com/adoptium/temurin-build/pull/3009), until the core issue is addressed.  To future-proof against similar problems, we will be adding smoke tests to be run in a stripped down environment.
+
+ 
+## Respins
+We have covered the issues introduced in our repositories that caused minor delays, now let's look at changes in the upstream OpenJDK repositories that resulted in the need to respin every JDK version of this release, a costly activity. 
+
+For the JDK8u345 respin, the issue is described in [JDK-8194154](https://bugs.openjdk.org/browse/JDK-8194154), which describes the crash that would occur when changing the "user.dir" property.  While changing user.dir is discouraged, some applications still do it, notably Gradle.  This is an interesting dilemna, where the feature is highly discouraged, but still possible, and therefore it is used in the field by some number of applications.  This highlights the fact that we need to be testing with a broad set of external applications, especially ones that are used widely.  We are actively incorporating a set of tests in the AQAvit suite for this purpose, but need to progress this work.  
+
+JDK11.0.16.1, JDK17.0.4.1 and JDK18.0.2.1: c2 regression (excessive memory consumption after 17.0.4 update): [JDK-8292260](https://bugs.openjdk.org/browse/JDK-8292260) - This defect caused *intermittent* crashes and was introduced as part of a CVE patch.  Since our Eclipse Foundation project is not a member of the Vulnerability Group (VG), we do not have access to these patches ahead of the GA tags being available.  
```

Comment:
Same here. Add a space between JDK and version: `JDK 11.0.16.1` etc.

Also: The analysis of this one is not correct. The respin was necessary due to a defect in a c2 releated patch that was included with JDK early access builds `17.0.4+1` and onwards and `11.0.16+1` and better. See https://bugs.openjdk.org/browse/JDK-8279219 Unfortunately it got unnoticed with the existing testing. Symptom was described in https://bugs.openjdk.org/browse/JDK-8291665 and was mainly "observable" on JDK 11 (less so on JDK 17).

Reply:
thanks, I plan to incorporate the table we discussed internally which will address this section